packages/react-scripts/template/README.md

```diff
@@ -566,6 +566,7 @@ Import Bootstrap CSS and optionally Bootstrap theme CSS in the ```src/index.js``
 import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap/dist/css/bootstrap-theme.css';
 ```
+Ensure the bootstrap css is imported before the rest of your app (i.e. before `import Routes from './routes';`) so that your styles are loaded afterwards.
```

Comment:
Yea I'm not that keen either, but as you say without explaining how styles work I'm not sure how better to do it. We could alternatively just put the entire contents of the (default) `index.js` in there, and remove the note?